docs/code_owners.md

```diff
@@ -12,14 +12,13 @@ and Clang's
 
 ### All parts not covered by anyone else
 
-- Sean Silva (@silvasean)
-- Stella Laurenzo (@stellaraccident) -- mostly emeritus
+- Stella Laurenzo (@stellaraccident)
```

Comment:
maybe leave me as emeritus?

Reply:
Landed before I got a chance: Updated in https://github.com/llvm/torch-mlir/commit/8ffe5d17da649591387dd907ff737bfa23bf6a79